ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_fft.py

```diff
@@ -243,17 +256,26 @@ def test_jax_numpy_rfft(
     dtype_input_axis, n, norm, frontend, backend_fw, test_flags, fn_tree, on_device
 ):
     input_dtype, x, axis = dtype_input_axis
+
     helpers.test_frontend_function(
         input_dtypes=input_dtype,
         frontend=frontend,
         backend_to_test=backend_fw,
         test_flags=test_flags,
         fn_tree=fn_tree,
         on_device=on_device,
+
```

Comment:
These changes are not related to the function you added, so you could undo these changes.